examples/android-gradle-kts/sentry.properties

```diff
@@ -0,0 +1,3 @@
+defaults.project=your-project
+defaults.org=your-org
+auth.token=YOUR_AUTH_TOKEN
```

Comment:
maybe instead of using a token, we could use the dryRun which is setting `autoUpload = false`
I'll check if I can set a token using Github Actions secret, I need to check with the security team.
cc @bruno-garcia

Reply:
you could install the CLI locally and authenticate using https://docs.sentry.io/product/cli/configuration/

Reply:
@cortinico I've created the `SENTRY_AUTH_TOKEN` GH repository secret, we'll need to change the value of `auth.token` before executing the tests